extensions/amp-sticky-ad/1.0/test/test-amp-sticky-ad.js

```diff
@@ -310,9 +310,7 @@ describes.realWin('amp-sticky-ad 1.0 with real ad child', {
     addToFixedLayerPromise = Promise.resolve();
     sandbox.stub(impl.viewport_, 'addToFixedLayer').callsFake(
         () => addToFixedLayerPromise);
-    return ampAd.implementation_.upgradeCallback().then(() => {
-      done();
-    });
+    return ampAd.implementation_.upgradeCallback();
```

Comment:
So is the new best practice just to return Promises and no longer to use done? Or does it not matter so long as it's consistent?

Reply:
Oh nvm. It doesn't matter so long as you're consistent. This change is just because this throws an exception now. Sorry--should have clicked through to the attached links.

Reply:
Yep. If you're returning a Promise, the call to `done()` used to be ignored, but it now flags an error.